langchain/callbacks/manager.py

```diff
@@ -28,6 +32,34 @@
 
 Callbacks = Optional[Union[List[BaseCallbackHandler], BaseCallbackManager]]
 
+openai_callback_var: ContextVar[Optional[OpenAICallbackHandler]] = ContextVar(
+    "openai_callback", default=None
+)
+tracing_callback_var: ContextVar[Optional[LangChainTracer]] = ContextVar(
+    "tracing_callback", default=None
+)
+
+
+@contextmanager
+def get_openai_callback() -> Generator[OpenAICallbackHandler, None, None]:
```

Comment:
Would it make sense to name all the context managers in a similar way? Eg. Starting with a common prefix. Otherwise I find it hard to know what is a context manager and what is just a function

Reply:
yeah need a better name for this... this is the way it was named before so I kept it that way to bw-compat reasons